backend/src/main/java/com/woowacourse/moragora/application/AttendanceService.java

```diff
@@ -109,10 +115,22 @@ public void updateAttendance(final Long meetingId,
     }
 
     public CoffeeStatsResponse countUsableCoffeeStack(final Long meetingId) {
-        final MeetingAttendances meetingAttendances = findMeetingAttendancesBy(meetingId);
-        validateEnoughTardyCountToDisable(meetingAttendances);
-        final Map<User, Long> userCoffeeStats = meetingAttendances.countUsableAttendancesPerUsers();
-        return CoffeeStatsResponse.from(userCoffeeStats);
+        validateMeetingExist(meetingId);
+        final List<Participant> participants = participantRepository.findByMeetingId(meetingId);
+        final int numberOfParticipants = participants.size();
+
+        final PageRequest pageRequest = PageRequest.ofSize(numberOfParticipants);
+        final List<Attendance> attendances = coffeeStatRepository.findCoffeeStatsLimitParticipant(
```

Comment:
```suggestion
        final List<Attendance> attendancesForCoffeeStack = coffeeStatRepository.findCoffeeStatsLimitParticipant(
```

이 값은 단순한 `출석 목록`이 아니라 `커피 스택`으로 쓰이게 될 data라는 것이 변수를 통해 전달되면 좋겠어요.
이거다 싶은 이름은 잘 떠오르지 않네요,,,

Reply:
attendancesForCoffeeStack 좋은 것 같아요!